contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -168,6 +168,9 @@
     private final List<Closeable> closeables = Lists.newArrayList();
     private final ExecutorService executorService;
     private final boolean enableStoreCreation;
+    // Executor to do store/schema verification in parallel
+    private final int THREAD_NUM_FOR_STORE_VERIFICATION = 20;
```

Comment:
Can this be made, configurable from the job.properties ?

If Hadoop has limtiation on the thread handles, this way we can work around for smaller time with config parameter.